src/main/webapp/app/components/tabs/ReviewHistoryTab.tsx

```diff
@@ -28,14 +31,14 @@ function ReviewHistoryTab({ isGermline, firebaseDb, drugList, getDrugs }: IRevie
   }, []);
 
   useEffect(() => {
-    getDrugs({ page: 0, size: GET_ALL_DRUGS_PAGE_SIZE, sort: ['id,asc'] });
+    getDrugs?.({ page: 0, size: GET_ALL_DRUGS_PAGE_SIZE, sort: ['id,asc'] });
   }, []);
 
   function handleDownload() {
-    const start = startDate ? new Date(startDate) : null;
-    const end = endDate ? new Date(endDate) : null;
+    const start = startDate ? new Date(startDate) : undefined;
```

Comment:
This is an example of a misuse of the `Partial` type. In this case, `StoreProps` should be changed to `ReturnType<typeof mapStoreToProps>`. Then the type system won't complain about things such as `getDrugs` that should never be null